src/daemon/scheduler.rs

```diff
@@ -127,7 +127,7 @@ fn make_cas_event_triggers(
                             Err(e) => error!("Unable to obtain CA list: {}", e),
                             Ok(list) => {
                                 for ca in list.cas() {
-                                    if publisher.publish(ca.handle(), &actor).await.is_err() {
+                                    if publisher.publish(ca.handle()).await.is_err() {
```

Comment:
Why is actor no longer passed here?

Reply:
Ah, it was event based before and so an actor was propagated to log as the initiator of the event.